chardet/sbcharsetprober.py

```diff
@@ -36,7 +36,7 @@ class SingleByteCharSetProber(CharSetProber):
     SB_ENOUGH_REL_THRESHOLD = 1024
     POSITIVE_SHORTCUT_THRESHOLD = 0.95
     NEGATIVE_SHORTCUT_THRESHOLD = 0.05
-    SYMBOL_CAT_ORDER = 250
+    SYMBOL_CAT_ORDER = 254
```

Comment:
I need it to distinguish between iso-8859-2 (latin2) and cp1250 (windows-1250) for some central european languages. It is hard to explain in one sentence, but I can try:
I need to take account (in the sbcharsetprober.py: self._total_char += 1) some CE letters (like šžťľ) which are letters in the latin2 but are symbols in the cp1250. If my solution is not good, I can try to (manually) decrease value in charmap tables of the CE language models from 253 to 252. Exactly said: The first and the second solution are not perfect, but without this I can't distinguish (big thanks M$) between latin2 and cp1250.